modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/BlogPosts/create.js

```diff
@@ -257,14 +257,15 @@ $(function () {
             initEditor(item);
         });
     }
-
+    
+    var editor;
```

Comment:
we can create an array for editors because a page can contain more than one editor. In that case, this code works for only the last initialized editor.